release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -593,6 +593,26 @@ With this release, the Agent-based Installer supports creating assets that can b
 
 For more information, see xref:../installing/installing_with_agent_based_installer/installing-using-iscsi.adoc#installing-using-iscsi[Preparing installation assets for iSCSI booting].
 
+[id="ocp-4.18-postinstallation-configuration_{context}"]
+=== Postinstallation configuration
```

Comment:
@sr1kar99 This is adding a new postinstall section (out of alphabetical order), and there is already a postinstall section below. You should move your stuff down below